aexpect/remote.py

```diff
@@ -372,20 +373,22 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
                     client(specify interface ip)
     :param preferred_authenticaton: Given value of PreferredAuthentications in ssh
     :param user_known_hosts_file: Given value of UserKnownHostsFile in ssh
+    :param extra_params: Additional params for the session, like -X for SSH.
```

Comment:
Maybe it is better to clarify the type at least implicitly here by saying "Additional params list for the session, like ['-X'] for SSH.